pkgs/by-name/as/asfa/package.nix

```diff
@@ -0,0 +1,59 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  pkgs,
+  ...
+}: let
+  version = "0.10.0";
+in
+  rustPlatform.buildRustPackage {
+    pname = "asfa";
+    inherit version;
+
+    src = fetchFromGitHub {
+      owner = "obreitwi";
+      repo = "asfa";
+      rev = "v${version}";
+      hash = "sha256-MnhnwtZmPFhOuiqNiaxJnPu88JOdlpvyVy0YGphblBc=";
+    };
+
+    cargoHash = "sha256-/bRBP/NzcNOXl/nANeOYouUAo3NNbtbV9fxIJrNajYQ=";
+
+    buildInputs = [
+      pkgs.libssh2
```

Comment:
I saw that you recently added this dependency to the upstream repo flake as well. The package seems to build without this.
Could you provide the reason behind why this is needed?

Reply:
Same as above, it was needed to get [crane](https://github.com/ipetkov/crane)-integration working. But I will have to recheck.

After manual verification, it's definitely not needed for this PR… :+1: